modules/oe_content_event/oe_content_event.post_update.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Content Event module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+
+/**
+ * Change widget for "Event type" field to skos select in form display.
+ */
+function oe_content_event_post_update_00001() {
+  $display = EntityFormDisplay::load('node.oe_event.default');
+  $content = $display->get('content');
+
+  if (empty($content['oe_event_type'])) {
+    return 'Event type field is not exposed on the form display.';
```

Comment:
Seeing that we are doing an update path here in the component, it means we are fixing something everywhere. So how come we are only fixing the default entity form display? What if there are more form displays that use the old select widget?

Reply:
We could also only do this on EWCMS, we have followed this rationale in other occasions too. @brummbar what do you think?

Reply:
This doesn't fix a bug indeed, so we should not have an opinion anymore on what's the best widget for the user that installed this module. Let's move this in EWPP.